### PR TITLE
Add mention of LIS2 and its fix for file size overflow.

### PR DESCRIPTION
**Pull request by metalex201**
I hope this fits in the required format of this doccumentation, sadly i cant provide a chinesse translation so that version would be behind in this addition.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,6 +313,12 @@ NOTE: I've been in contact with someone at Google who passed on this message fro
 23. We send CLSE to device
 24. Device sends us CLSE
 
+NOTE: On files over the integer limit (around 2.14 GB) the 4 byte size will overflow resulting in a 2.14GB file recounting from 0 This issue is fixed within the code of the AOSP ADB implementation by using 8 bytes instead of 4 (whitout any mention of it in the AOSP's own doccumentation 🙃) this works the same way as the before mentioned protocol for adb list the change is on step 5 instead of LIST we send LIS2, and instead of DENT the device will send back DENT2 and the received packet will be in a different sized format, heres an example listing the differences between LIST and LIS2 (notice the size in part 3. being the correct size of 9.9GB )
+
+![host](/images/DENT2.png)
+
+After DEN2 there is 20 bytes of unknown data ( could be some extra file info that is undocumented ) after that step 2., 3., 4., are the same as the before mentioned list of file data except its 8 bytes long instead of 4 and theres an extra added 8 bytes (which is undoccumented ) parts 5., 6. are exactly the same.
```
Comment (cstyan) [inline README.md:320]:
```suggestion
After DENT2 there is 20 bytes of unknown data ( could be some extra file info that is undocumented ) after that step 2., 3., 4., are the same as the before mentioned list of file data except its 8 bytes long instead of 4 and theres an extra added 8 bytes (which is undoccumented ) parts 5., 6. are exactly the same.
```

File: README.md [hunk 2]
```diff
@@ -313,6 +313,12 @@ NOTE: I've been in contact with someone at Google who passed on this message fro
 23. We send CLSE to device
 24. Device sends us CLSE
 
+NOTE: On files over the integer limit (around 2.14 GB) the 4 byte size will overflow resulting in a 2.14GB file recounting from 0 This issue is fixed within the code of the AOSP ADB implementation by using 8 bytes instead of 4 (whitout any mention of it in the AOSP's own doccumentation 🙃) this works the same way as the before mentioned protocol for adb list the change is on step 5 instead of LIST we send LIS2, and instead of DENT the device will send back DENT2 and the received packet will be in a different sized format, heres an example listing the differences between LIST and LIS2 (notice the size in part 3. being the correct size of 9.9GB )
```
Comment (cstyan) [inline README.md:316]:
```suggestion
NOTE: On files over the integer limit (around 2.14 GB) the 4 byte size will overflow resulting in a 2.14GB file recounting from 0 This issue is fixed within the code of the AOSP ADB implementation by using 8 bytes instead of 4 (whitout any mention of it in the AOSP's own doccumentation 🙃). This works the same way as the before mentioned protocol for the list command, except the change is on step 5, instead of LIST we send LIS2 and instead of DENT the device will send back DENT2. The received packet will be in a different sized format, here's an example listing the differences between LIST and LIS2 (notice the size in part 3. being the correct size of 9.9GB )
```

File: images/freecad.lnk
Comment (cstyan) [inline images/freecad.lnk:1]: I don't think you meant to include this file?

File: README.md [hunk 3]
```diff
@@ -97,9 +97,9 @@ Here's what the AOSP's own implementation of ADB does:
 ![cnxn](https://github.com/cstyan/adbDocumentation/raw/master/images/googleCNXN.jpg)  
 Notice that the 8 bytes are the same as the bytes previous to the `host::` bytes 
 in the last hex dump. This is from the `data_length` and `data_crc32` fields 
-being set based on wanting to send `host::` as our data.    
-2. The `host::` system-identity string
-![host](https://github.com/cstyan/adbDocumentation/raw/master/images/googleHost.png)  
+being set based on wanting to send `host::` as our data.
+2. The `host::` system-identity string                       
+   ![host](https://github.com/cstyan/adbDocumentation/raw/master/images/googleHost.png)
```
Comment (cstyan) [inline README.md:102]: unless this changes the rendered formatting in a way that you think is useful, lets avoid making unrelated whitespace changes here

**Review by cstyan — COMMENTED** [review]
just a few minor comments

File: images/freecad.lnk
Comment (metalex201) [reply images/freecad.lnk:1]: oops sorry for that.

File: README.md (hunk 3 above)
Comment (metalex201) [reply README.md]: if the text is viewed in fullscreen the image gets put to the right of the text, altought this was only visible in the editor? I'm sorry i dont use github often so i just thought i did something wrong for it to look like that.
image

File: README.md (hunk 1 above)
Comment (metalex201) [reply README.md:320]: I completely missed over this, the size for the start message is 4 characters everytime but its not DEN2 is DNT2

File: README.md (hunk 3 above)
Comment (cstyan) [reply README.md]: If we view the rendered form of the markdown as it currently exists in the master branch, the formatting is correct
2024-03-28-103026_1325x331_scrot


But if we view the rendered markdown for your PR the formatting of the image puts it on the same line as you've shown
2024-03-28-103049_1523x492_scrot

So we should still be fine just to remove the whitespace changes here.

File: README.md (hunk 2 above)
Comment (cstyan) [reply README.md:316]: @metalex201 the changes I'm suggesting here are still relevant, especially since there's a `DENT2` in here, but you're saying it should be `DEN2`.

Comment (metalex201) [reply README.md:316]: it should be` DNT2`

Comment (metalex201) [reply README.md:316]: I'm gonna commit changes and make another pull request with an image update (because it also has DENT2 instead of DNT2) and a change from `DENT2` to `DNT2`

File: README.md [hunk 4]
```diff
@@ -97,9 +97,9 @@ Here's what the AOSP's own implementation of ADB does:
 ![cnxn](https://github.com/cstyan/adbDocumentation/raw/master/images/googleCNXN.jpg)  
 Notice that the 8 bytes are the same as the bytes previous to the `host::` bytes 
 in the last hex dump. This is from the `data_length` and `data_crc32` fields 
-being set based on wanting to send `host::` as our data.    
+being set based on wanting to send `host::` as our data.
 2. The `host::` system-identity string
-![host](https://github.com/cstyan/adbDocumentation/raw/master/images/googleHost.png)
```
Comment (cstyan) [inline README.md:102]: the whitespace changes here should still be removed

**Comment by cstyan** [comment]
The freecad image is back, and you can change the name of the image file here instead of via a new PR :)

**Comment by metalex201** [comment]
great! Now if i didnt miss anything it should be okay to merge.

File: README.md (hunk 4 above)
Comment (cstyan) [reply README.md:102]: these are still here but I'll just fix them myself after merging
